src/cheri_ptw.sail

```diff
@@ -24,32 +24,41 @@ function ptw_error_to_str(e) =
 
 overload to_str = {ptw_error_to_str}
 
-function ext_get_ptw_error(ext_ptw : ext_ptw) -> PTW_Error =
-  match (ext_ptw) {
-    PTW_CAP_OK        => PTW_No_Permission(),
-    PTW_LOAD_CAP_ERR  => PTW_Ext_Error(AT_CAP_ERR),
-    PTW_STORE_CAP_ERR => PTW_Ext_Error(AT_CAP_ERR)
+/*
+ * Produce the PTW_Error for PTE_Check_Failure-s arising from
+ * checkPTEPermission calls.
+ */
```

Comment:
I'm not sure this comment is even necessary now? The use of the extra type makes things much clearer to follow. It feels a lot like `/* Get the foo */ Foo getFoo()` now.